node/src/main/scala/org/bitcoins/node/networking/peer/DataMessageHandler.scala

```diff
@@ -107,7 +107,7 @@ case class DataMessageHandler(
                   peerMsgSender,
                   startHeightOpt).map { synced =>
                   if (!synced) logger.info("We are synced")
-                  syncing
+                  synced
```

Comment:
should this be `!synced`? `!synced` would imply to me that we are `syncing`

Reply:
The correct variable name should be `syncing` so i rename it. It's unfortunate that we also have a `syncing` value passed as a parameter to `DataMessageHandler` too which can cause confusion. Hopefully I can get rid of that in a future PR.

https://github.com/bitcoin-s/bitcoin-s/blob/c4d358061a20312cd30b252c853b4536e51a20d3/node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageSender.scala#L213